docs/user-guide/indexing.rst

```diff
@@ -0,0 +1,31 @@
+.. _usage-indexing:
+
+Indexing
+========
+
+Hist implements the UHI indexing protocol. You can read more about it on the `UHI Indexing <https://uhi.readthedocs.io/en/latest/indexing.html>`_ page.
```

Comment:
We could also link to the UHI+ page that follows, since Hist follows that one too.

Reply:
Okay